release-scripts/do_release.py

```diff
@@ -91,6 +91,18 @@
     (re.compile(r'__version__\s*=\s*".*"'),
      '__version__ = "{version-no-v}-dev"'),
 
+    (re.compile(r'__libnetwork_plugin_version__\s*=\s*".*"'),
```

Comment:
Not sure it's quite this simple.  At the moment we've broken the old behavior of having master calico-docker point at latest libnetwork-plugin and latest calico/node.  But can't simply set to latest because it will break the behavior of defaulting the versions to the versions in the last release.